challenges/palindrome-detector.js

```diff
@@ -21,3 +21,32 @@
 */
 
 // YOUR CODE HERE
+
+function isPalindrome(userInput) {
+  var  arrayInput = userInput.split('');
+
+  let toStop = false;
+
+  while (!toStop) {
+    toStop = true;
```

Comment:
This looks like it's very much based on the sorting code from yesterday, and the outer loop isn't necessary; it will only ever run once. So if you got rid of lines 28, 29, 30, 31, and 43, your code would run in exactly the same way.